jaxrs/pom.xml

```diff
@@ -53,13 +69,12 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>javax.ws.rs-api</artifactId>
         </dependency>
-
-        <!--Test Dependencies-->
         <dependency>
             <groupId>org.jboss.resteasy</groupId>
             <artifactId>resteasy-jaxrs</artifactId>
-            <scope>test</scope>
```

Comment:
Since this is a generic jaxrs module, I'm not sure if we want to pull in resteasy libraries. Maybe a jaxrs-resteasy module?

Reply:
Since this dependency was already here in the test scope for other classes outside the changes for this story, I have moved the FailureExceptionMapper to the `beadledom-resteasy` project: https://github.com/cerner/beadledom/pull/45/commits/ff9039283a54bd60c160857d65055c3ba9d1b35f and moved this dependency back under `Test Dependencies`: https://github.com/cerner/beadledom/pull/45/commits/39df379bb334bb6bda3aa897056d2f9a12a21b52